extension_config.cmake

```diff
@@ -2,8 +2,8 @@
 
 duckdb_extension_load(avro
         LOAD_TESTS
-        GIT_URL https://github.com/duckdb/duckdb_avro
-        GIT_TAG aa2741544c5b53cc8fda6a85ca56bd0353116de4
+        GIT_URL https://github.com/tishj/duckdb_avro
+        GIT_TAG 9735db41b8fa1d409eb4d0e20e3e16499da503f9
```

Comment:
This should point at `duckdb/duckdb_avro`, for that the PR needs to be merged there first